### ISSUE TITLE
Ajouter une licence

### ISSUE THREAD

**Issue by bzg** [etalab/schema-irve#31]
Je suppose que la licence Ouverte 2.0 est la licence pour les schémas, mais vous savez mieux.

**Comment by thbar** [comment]
C'est une bonne question. Je ne sais pas si le schéma est corrélé à un usage de licence précis en tant que tel, ni si une disposition juridique l'impose.

Je propose d'en parler, poke @jmaupetit @stephane-pignal et Nicolas Forget.

**Comment by bzg** [comment]
Des nouvelles? Gentle poke :)

**Comment by thbar** [comment]
Merci @bzg pour la relance ! Sujet non traité pour l'instant. Je vais me renseigner.

**Comment by bzg** [comment]
Hello, je me permets une relance :)

**Comment by thbar** [comment]
@bzg peux-tu nous donner du contexte sur ton attente ? 

Par ailleurs, tu parles bien de la license du schéma lui-même, ou de celle de la donnée ?

Merci !

**Comment by bzg** [comment]
Oui, il s'agit de la licence du schema que vous proposez dans le dépôt, donc une licence Ouverte 2.0 ("Etalab") ira très bien.

**Comment by thbar** [comment]
Pour l'instant on n'est pas en mesure de statuer sur ce point (et je ne souhaite pas prendre une décision non éclairée, d'expérience sur ce sujet, pour le moment). J'ai demandé conseil mais "don't hold your breath" pour le moment.

**Comment by thbar** [comment]
Note de @Pierlou que je remercie:

> Je suppose que ce n'était pas encore le cas à l'époque de la création du schéma IRVE, mais nous proposons dans [notre template de création de schéma](https://github.com/datagouv/tableschema-template) la licence LO2 par défaut, qui est intégrée au repo

Je vais partir sur LO2, mais mettre un email à chaque contributeur quand même pour le principe.

**Comment by bzg** [comment]
<img width="700" height="690" alt="Image" src="https://github.com/user-attachments/assets/d6f1cd66-4f56-4702-814f-621f577aee19" />

**Comment by thbar** [comment]
On est en budget tendu et en priorisation assez hardcore, avec pas mal de sujets ops à traiter, alors je ne vais pas traiter pour le moment.

On verra si on a la bande passante pour faire ça (je sais que c'est pas grand chose, mais beaucoup de sujets d'importance équivalente) un peu plus tard.

Je clôture pour le moment du coup.

**Comment by bzg** [comment]
Oui, pas de souci - merci !
